components/experimental/src/lib.rs

```diff
@@ -53,6 +53,11 @@ pub mod provider {
         impl_currency_patterns_data_v1!(Baked);
         impl_currency_extended_data_v1!(Baked);
         impl_units_display_name_v1!(Baked);
+        impl_area_display_name_v1!(Baked);
+        impl_duration_display_name_v1!(Baked);
+        impl_length_display_name_v1!(Baked);
+        impl_mass_display_name_v1!(Baked);
+        impl_volume_display_name_v1!(Baked);
```

Comment:
Nit: You're adding data without removing data. Please make this PR be data-neutral.

Reply:
Sure, I have added an issue to track it : https://github.com/unicode-org/icu4x/issues/6900